docs/source/admin/quick_howto/server_capabililty.rst

```diff
@@ -42,18 +42,18 @@ Users with the Operations or Admin :term:`Role` can associate one or more server
 
 .. _delivery_service_required_server_capabilities:
 
-Manage delivery service required server capabilities
-====================================================
-Users with the Operations or Admin :term:`Role` can associate one or more required server capabilities with a :term:`Delivery Service` by navigating to a :term:`Delivery Service` via :menuselection:`Services --> Delivery Services` and using the context menu for the :term:`Delivery Services` table and selecting :menuselection:`Manage Required Server Capabilities` or by navigating to :menuselection:`Services --> Delivery Services --> Delivery Service --> More --> Manage Required Server Capabilities`.
+Configure delivery service required server capabilities
+=======================================================
+Users with the Operations or Admin :term:`Role` can associate one or more required server capabilities with a :term:`Delivery Service` by navigating to a :term:`Delivery Service` via :menuselection:`Services --> Delivery Services` and clicking on Required Capabilities checkbox present on Delivery Services form (HTTPS and DNS)`.
```

Comment:
actually this part changed. it should say "Users with the DELIVERY-SERVICE:CREATE or DELIVERY-SERVICE:UPDATE permission can associate..."